src/features/Banner/component.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable max-len */
+import React, { Component } from 'react';
+// import components
+import Navbar from '../../shared/Navbar';
+import SearchMovie from '../../shared/SearchMovie';
+import './style.scss';
+
+export default class Banner extends Component 
+{
+	constructor(props){
```

Comment:
in latest react empty constructor is redundant

![Screenshot 2021-09-28 at 14 43 30](https://user-images.githubusercontent.com/25443181/135098894-542033c1-0b13-4c5d-ae4e-9993b1b4560f.png)

Reply:
These are removed